src/main/java/pt/teamloan/service/PostingsService.java

```diff
@@ -231,6 +247,18 @@ private void mapJobId(PostingJobEntity postingJob) throws TeamLoanException {
 		}
 	}
 
+	private void setPostingStatus(PostingEntity postingEntity, PostingStatus status) throws TeamLoanException{
+		postingEntity.setStatus(status);
+		postingEntity.setUpdatedStatusAt(Timestamp.from(Instant.now()));
+	}
+
+	private PostingStatus mapPostingStatus(ClosePostingReason closePostingReason) throws TeamLoanException{
+		if(closePostingReason == ClosePostingReason.MATCH){
```

Comment:
Always prefer using `equals` to `==`.
In this case it would be 
```java
if (ClosePostingReason.MATCH.equals(closePostingReason))
```
order is important to avoid `NullPointerException`

In this case it does not make much of a difference because enums are singletons so the reference to the instance will actually be the same. But a good argument is that the `equals` function can be implemented on the enum and it wouldn't change the behaviour on this `if` statement.